tests/check_ipc.c

```diff
@@ -64,7 +64,7 @@ static const int MAX_MSG_SIZE = DEFAULT_MAX_MSG_SIZE;
 #define GIANT_MSG_DATA_SIZE MAX_MSG_SIZE - sizeof(struct qb_ipc_response_header) - 8
 
 /* This can take AGES (like, an hour) on FreeBSD */
-#ifdef __FreeBSD__
+#ifndef HAVE_SLOW_TESTS
```

Comment:
I think we should relax testing being in place for several years already
only on demand, through that sketched `--disable-sticky-tests` equivalent.
The hypothesis that it's _always_ slow on FreeBSD has been invalidated,
so only a very very specific subset of all setups is affected (VMs with
FreeBSD lacking virtio support under KVM?).